evm-ds/src/protos/evm_proto.rs

```diff
@@ -65,6 +65,7 @@ pub struct EvmCallArgs {
     pub enable_cps: bool,
     pub tx_trace_enabled: bool,
     pub tx_trace: Arc<Mutex<LoggingEventListener>>,
+    pub is_scilla: bool,
```

Comment:
In terms of imports, scilla refers in limited cases to evm or evm-ds, but not to zilliqa. This is to avoid circular dependencies